CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Current Develop Branch
 
 ## 7.7.6 Mon Mar 02 2020
+- [#8154](https://github.com/MetaMask/metamask-extension/pull/8154): Use @metamask/eth-ledger-bridge-keyring@0.2.2
```

Comment:
Perhaps this would be better described as a bug fix, rather than a routine dependency update?
e.g. something like
```suggestion
- [#8154](https://github.com/MetaMask/metamask-extension/pull/8154): Prevent signing from incorrect Ledger account
```